pypanther/rules/github/github_repo_vulnerability_dismissed.py

```diff
@@ -24,15 +24,6 @@ def alert_context(self, event):
         return context | {"alert_url": alert_url}
 
     tests = [
-        RuleTest(
```

Comment:
Applying the change seen here: https://github.com/panther-labs/panther-analysis/pull/1569/files